cmd/restart.go

```diff
@@ -23,7 +23,7 @@ var (
 	  scalingo --app my-app restart web-1`,
 		Action: func(c *cli.Context) {
 			currentApp := appdetect.CurrentApp(c)
-			if err := apps.Restart(currentApp, c.Bool("synchronous"), c.Args()); err != nil {
+			if err := apps.Restart(currentApp, c.Bool("synchronous") || c.Bool("s"), c.Args()); err != nil {
```

Comment:
Not necessary, both forms can't be present in the command line, and the same value is set for the long and short flag.